### DISCUSSION TITLE
"this day reminder" notification

### DISCUSSION THREAD

**Discussion by cazzoo** [Radiokot/photoprism-android-client#143] [Ideas]
I was using amazon photos app for a while but i recently migrated to a self hosted photoprism solution. Now discovered the gallery app, which is really neat but i miss one feature which is daily notifications of my past pictures for the day of the notification.
Let say today 15/07/2024 i get a notification for all the photos i took on 15th of Jully.

Also there is a selection of random surrounding photos for one year as a short movie.

What do you think about it in this app?

**Comment by Radiokot** [comment]
Hey, glad you moved to PhotoPrism and like this app 😎

The feature you are looking for is already available in the gallery as an extension called **Memories**. It brings you a daily collection of photos and videos from the same day in past years. The gallery searches for memories once a day in the morning. In the extension preferences, you can enable sending a notification whenever new memories are found.
<p> 
<img src="https://github.com/Radiokot/photoprism-android-client/assets/5675681/1c1ad338-da36-4023-84a8-77dd2b6b5e25" width=320/>
<img src="https://github.com/Radiokot/photoprism-android-client/assets/5675681/349dde1a-d22f-45bc-9e75-be91c0ba1eea" width=231 />
</p>

Gallery extensions are extra features of the app which could be either purchased individually, each for its own small price, or acquired for free by sponsors and contributors. Extensions are activated by perpetual offline license keys – no accounts and subscriptions required.

Memories are **2$** and free for sponsors and contributors. I [accept](https://radiokot.com.ua/tip) Bitcoin, USDT, Monero or PayPal. If you'd like to purchase the extension, please, go to Preferences -> Extensions in the gallery and click the Help button to contact me by email 🙌🏻

**Reply by cazzoo** [reply]
Lovely, thanks
